src/seamless_communication/inference/transcriber.py

```diff
@@ -295,20 +308,34 @@ def transcribe(
             Window size to pad weights tensor.
         :params **sequence_generator_options:
             See BeamSearchSeq2SeqGenerator.
+        :params denoise:
+            Whether to denoise the audio.
+        :params denoise_model:
```

Comment:
nit: we don't need them anymore, right?

Reply:
That's right, but I'm keeping the denoise bool so that denoising can be used without creating a DenoisingConfig object